.github/workflows/flow-temp.yml

```diff
@@ -9,19 +9,12 @@ name: temporary testing
 
 on:
   push:
-    branches-ignore: ['**'] # ignore all branches. Comment this line to run your workflow below on every push.
+   # branches-ignore: ['**'] # ignore all branches. Comment this line to run your workflow below on every push.
```

Comment:
there is a commit commenting this out,
I'll push it when the currentrun finishes